### PR TITLE
Info on how to provide the files for a new kernel

### PR DESCRIPTION
**Pull request by timothymcmackin**
This page has info about how to provide the hash of a proposed kernel, but for completeness, this PR adds a little info about providing the code of the kernel for people to evaluate and the preimages of the kernel for people to upgrade to.

### CODE REVIEW

File: docs/governance/how-do-i-participate-in-governance.md [hunk 1]
```diff
@@ -128,6 +131,9 @@ octez-client transfer 0 from tz1RfbwbXjE8UaRLLjZjUyxbj4KCxibTp9xN to KT1HfJb718f
   --arg "\"sr1EStimadnRRA3vnjpWV1RwNAsDbM3JaDt6\""
 ```
 
+After the new kernel becomes active, bakers must upgrade their nodes to it to continue to participate in Etherlink.
```
Comment (lthms) [inline docs/governance/how-do-i-participate-in-governance.md:134]: I’m not sure how, but we should reword here. We don’t need to upgrade nodes (as in, upgrading to a more recent version), we need to provision them with the correct preimages.

Comment (timothymcmackin) [reply docs/governance/how-do-i-participate-in-governance.md]: How about this?

> After the new kernel becomes active, bakers must provide their nodes with the preimages for the new kernel to continue to participate in Etherlink.

Comment (lthms) [reply docs/governance/how-do-i-participate-in-governance.md]: s/provide/provision?

Comment (timothymcmackin) [reply docs/governance/how-do-i-participate-in-governance.md]: What do you mean by "provision?" Do you mean that node operators would provision a new container or virtual image? I assumed that node operators could merely restart the node process when the preimages endpoint gets updated.

Comment (lthms) [reply docs/governance/how-do-i-participate-in-governance.md]: I’m using “provision” here without any specific technological implication, but rather as a way to say “make something available to”.

Here it’s simply copying the correct the files in the expected directory, no need to restart the node.

Anyway, I think your suggestion works as well.

Comment (timothymcmackin) [reply docs/governance/how-do-i-participate-in-governance.md]: Updated to:

> After the new kernel becomes active, bakers must provide their nodes with the preimages for the new kernel to continue to participate in Etherlink. They can copy the new preimages into the node data directory without stopping or restarting the node. If a node uses a preimages endpoint as described in [Running an Etherlink EVM node](http://localhost:3000/network/evm-nodes) and [Running an Etherlink Smart Rollup node](http://localhost:3000/network/smart-rollup-nodes), the node updates automatically.

**Comment by lthms** [comment]
LGTM.

**Pull request merged**